.gitignore

```diff
@@ -9,4 +9,11 @@
 .history/
 
 # Built Visual Studio Code Extensions
-*.vsix
```

Comment:
Don't put this here.  You can have your own `.gitignore` inside of the `elastio-fargate-stream-backup` directory where project-specific ignores go.